gem2rpm/scl.spec.erb

```diff
@@ -89,7 +89,7 @@ cp -a .%{gem_dir}/* \
 
 <% unless spec.extensions.empty? -%>
 mkdir -p %{buildroot}%{gem_extdir_mri}
-cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+cp -a .%{gem_instdir}/ext/%{gem_name}/*.so %{buildroot}%{gem_extdir_mri}/
```

Comment:
This no longer copies the `gem.build_complete` file which I think is an issue. Is the real issue that `%{gem_extdir_mri}` is actually incorrect? I think I've seen that under in a package that's build both SCL and non-SCL. I can't find the PR now, but rubygem-mysql is an example of that.

Reply:
Turns out, it was more simple than what I had. The `.so` file appears to end up in a directory up from the `gem.build_complete` and this was not copying both. I updated this line to catch the `.so` at the right spot.